From 141c75a602161eb2697c2941168f5fa69a4c8cfc Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Mon, 2 Apr 2018 17:42:43 +0800
Subject: [PATCH 3560/5242] MLK-17944: ARM: dts: imx7ulp-evk: add poweron key

commit  fae937c323f6e4e56ab33bbef0b7e2a1af6ab2ac from
https://source.codeaurora.org/external/imx/linux-imx.git

Add poweron key support on i.mx7ulp-evk board since M4 take
over snvs on B0 chip.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
(cherry picked from commit 6684b9cdef60053ebaa972b401842cc813d974d8)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 8aa9013..c9f6cf9 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -187,6 +187,12 @@
 			label = "Volume Down";
 			linux,code = <KEY_VOLUMEDOWN>;
 		};
+
+		power-on {
+			label = "PowerOn";
+			linux,code = <KEY_POWER>;
+			rpmsg-key,wakeup;
+		};
 	};
 
 	rpmsg_sensor: rpmsg-sensor {
-- 
1.7.9.5

